src/detectors/FHCAL/FHCAL.cc

```diff
@@ -245,8 +261,43 @@ extern "C" {
           "LFHCALTruthProtoClusters", {"LFHCALRecHits", "LFHCALHits"}, {"LFHCALTruthProtoClusters"},
           app   // TODO: Remove me once fixed
         ));
-        app->Add(new JChainFactoryGeneratorT<ProtoCluster_factory_LFHCALIslandProtoClusters>(
-          {"LFHCALRecHits"}, "LFHCALIslandProtoClusters"
+
+        // Magic constants:
+        //  24 - number of sectors
+        //  5  - number of towers per sector
+        //  moduleIDx
+        //  moduleIDy
+        //  towerx
+        //  towery
+        //  rlayerz
```

Comment:
Those comments must have been copied over from SciGlass. If I had to guess:
```suggestion
        //  54 - number of modules in a row/column
        //  2  - number of towers in a module
```
cc @FriederikeBock @steinber 
Makes me wonder why there is no 4 (if 8M modules are used).

Reply:
Suggestion to address any necessary logic changes in a separate PR :-)